server/data/restClient.ts

```diff
@@ -35,7 +36,7 @@ export default class RestClient {
 
   logger: Logger
 
-  constructor(private readonly name: string, private readonly config: ApiConfig, private readonly token: string) {
+  constructor(private readonly name: string, private readonly config: ApiConfig, private readonly token = '') {
```

Comment:
does this mean that if the caller of this doesn't specify token we explicitly send `Authorization: Bearer` with no token content?

would it be better to default to null and not set the auth header full stop?

Reply:
That's a good point, I guess the caller could not pass in the token on initialization or when calling `get` so we'd sent that - I could either throw an error on an empty string or throw one if it's null  - what do you think would be better?